transformer_lens/components/mlp.py

```diff
@@ -16,6 +16,19 @@
 from transformer_lens.utils import gelu_fast, gelu_new, solu
 
 
+def addmm(bias, weight, x):
```

Comment:
Could you actually move this into the class itself, in order to avoid potential name conflicts in the future?

Reply:
The reason I left it here is that I'm 85-90% sure that in the change I alluded to above ("there are several other places the same fix should probably be applied"), this will become a shared utility. From a quick glance I'd guess around half of the uses of `einsum` in the codebase should have this same fix applied. So we'd want to revert adding it to the class in the next change.

Reply:
That makes sense. If that is the case, honestly I would probably add a new utilities file where we can have these equations. It's a bit hidden away here for other people, so putting in a shared place to begin with would probably be a good idea.

Reply:
`utils.py` seems like the current dumping ground for this kind of thing?

Reply:
`utils.py` is currently being refactored into separate files inside of `utilities`. I believe there is a pending PR to review for this, so putting it into a separate file here will make that guys work load a bit less.